control/control.yaml

```diff
@@ -124,4 +124,16 @@ components:
             Array of ping requests
           type: array
           items:
-            $ref: './ping.yaml#/components/schemas/Ping'
\ No newline at end of file
+            $ref: './ping.yaml#/components/schemas/Ping'
+
+    Protocol.State:
+      description: >-
+        Sets all configured protocols to intended state
```

Comment:
started or stopped state instead of intended state ?

Reply:
changed to mention `start` or `stop` state.